src/objects/setup_configuration/steps/sites.py

```diff
@@ -0,0 +1,37 @@
+from django.contrib.sites.models import Site
+from django.core.exceptions import ValidationError
+from django.db import IntegrityError
+
+from django_setup_configuration.configuration import BaseConfigurationStep
+from django_setup_configuration.exceptions import ConfigurationRunFailed
+
+from objects.setup_configuration.models.sites import SitesConfigurationModel
+
+
+class SitesConfigurationStep(BaseConfigurationStep):
+    config_model = SitesConfigurationModel
+    verbose_name = "Sites configuration"
+
+    namespace = "sites"
+    enable_setting = "sites_config_enable"
```

Comment:
In this regard, I suggest having a unique naming convention for all projects and apps, so as not to create problems with different yaml files.
`%s_%s' % (app_label,  model_name)`
- something like how the url is made in admin
```suggestion
    namespace = "sites_site"
    enable_setting = "sites_site_config_enable"
```

Reply:
Good point, although the suggested namespace probably will clash whenever other projects allow their `Site` to be configured as well. Consider for instance the following configuration file:
```yaml
# Objects-api
sites_site_config_enable: true
sites_site:
  items:
    ....

# Objecttypes-api
sites_site_config_enable: true
sites_site:
  items:
    ....
```

I think prefixing it with the project name will fix this, for example `objects_api_sites_config` and `objects_api_sites_config_enable`.

Reply:
@danielmursa-dev I think the collision of the namespaces, like mentioned above, also applies to the token configuration?

Reply:
yes exactly, everywhere the same namespaces must be used

Reply:
> @danielmursa-dev I think the collision of the namespaces, like mentioned above, also applies to the token configuration for certain projects?

Discussed through the chat, this comment can be ignored as the namespace collision will not happen (different configuration file per project). The project prefix is not necessary for `Site` configuration.

Reply:
@SonnyBA OK , so let's continue with this format yes?
```
# %s_%s' % (app_label,  model_name)

sites_site_config_enable:` true
sites_site:
  items:
    ....
```

Reply:
I don't think it's necessary to change this in the steps in existing projects but this might be a good scheme for steps that are going to be added.